prototypes/htmx/src/avisi/apps/tech_testing_ground/prototypes/htmx/server.clj

```diff
@@ -1,16 +1,36 @@
 (ns avisi.apps.tech-testing-ground.prototypes.htmx.server
   (:require
-    [avisi.apps.tech-testing-ground.prototypes.htmx.hello-world :as hello-world]
+    [avisi.apps.tech-testing-ground.prototypes.htmx.item-views.jira-create-item-link-modal :as jira-create-item-link]
+    [avisi.apps.tech-testing-ground.prototypes.htmx.item-views.jira-item-view :as jira-item-view]
+    [avisi.apps.tech-testing-ground.prototypes.htmx.item-views.monday-create-item-link-modal :as
+     monday-create-item-link]
+    [avisi.apps.tech-testing-ground.prototypes.htmx.item-views.monday-item-view :as monday-item-view]
     [avisi.apps.tech-testing-ground.prototypes.shared.peripherals.server :as server]
     [mount.core :as mount :refer [defstate]]
     [ring.util.response :refer [redirect]]))
 
-(def routes (hello-world/routes))
+(def routes
+  [(jira-item-view/routes) (jira-create-item-link/routes) (monday-item-view/routes) (monday-create-item-link/routes)])
+
+(defn jira-req->domain-query-string [{{:strs [project-id issue-key]} :query-params}]
+  (str "?" "board-id=" project-id "&" "item-id=" issue-key))
+
+(defn monday-req->domain-query-string [{{:strs [boardId itemId]} :query-params}]
+  ; TODO: find out and fix monday sometimes sending query-params double in an array
```

Comment:
Een todo moet altijd een link hebben naar een issue, misschien kan die weg want je hebt het al opgelost. Dit is een bug en monday.com heeft dit al 6 maanden en nog altijd niet opgelost.